corehq/apps/reports/static/reports/javascripts/config.dataTables.bootstrap.js

```diff
@@ -9,6 +9,7 @@ function HQReportDataTables(options) {
     self.startAtRowNum = options.startAtRowNum || 0;
     self.showAllRowsOption = options.showAllRowsOption || false;
     self.aoColumns = options.aoColumns;
+    self.aoColumnDefs = options.aoColumnDefs;
```

Comment:
passing 'bootstrap' as the option for this is actually how this should be handled.